packages/database-connect/src/dynamicUserConnect.ts

```diff
@@ -131,6 +131,9 @@ export const connectToPostgresWithDynamicUser = (
         user,
         password,
         poolSize: 5,
+        connectionTimeoutMillis: 10000, // 10s: fail if can't connect
+        statement_timeout: 120000, // 120s: server fail if query takes longer
```

Comment:
What is the reasoning behind the timeout durations used here? A statement timeout only 20% shorter than the query timeout feels a bit redundant. No harm, just wondering why this specific config was chosen?